src/sections/WhatIsADebtorsUnion/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import Image from 'gatsby-image';
+import Button from 'components/Button';
+
+const WhatIsADebtorsUnion: React.FC = () => {
+  const data = useStaticQuery(graphql`
+    query {
+      coverImage: file(relativePath: { eq: "debtors-union-cover.png" }) {
+        childImageSharp {
+          fluid(quality: 100) {
+            ...GatsbyImageSharpFluid
+          }
+        }
+      }
+      debtorsUnionImage: file(relativePath: { eq: "debtors-union.png" }) {
+        childImageSharp {
+          fluid(fit: COVER) {
+            ...GatsbyImageSharpFluid
+            ...GatsbyImageSharpFluidLimitPresentationSize
+          }
+        }
+      }
+    }
+  `);
+
+  const coverImage = data?.coverImage?.childImageSharp?.fluid?.src || '';
+
+  return (
+    <section
+      className="min-h-screen flex flex-col justify-center bg-no-repeat bg-cover bg-center p-screen-spacing lg:px-desktop-screen-spacing"
+      style={{
+        backgroundImage: `url(${coverImage})`
+      }}
+    >
+      <div className="flex flex-col justify-between mb-12 lg:mb-24 lg:flex-row">
+        <div className="w-full order-2 lg:order-1 lg:w-1/2">
+          <h2 className="text-4xl leading-10 mt-8 text-black-100 font-black lg:leading-15 lg:text-5xl lg:mt-0">
+            What is a <br className="hidden lg:inline" /> Debtor’s Union?
+          </h2>
+          <p className="text-gray font-bold text-xl leading-6 mt-4">
+            In a workplace, workers come together to defend and fight for their
+            interests against greedy bosses and owners. Similarly, debtors are
+            coming together to form a union to defend our interests. We use our
+            collective power to fight for better conditions in our financial
+            lives and a more equitable society: the cancellation and
+            renegotiation of debts, access to publicly-funded goods (education,
+            healthcare, housing, etc), anti-racist economic policies, and much
+            more.
+          </p>
+        </div>
+        <Image
```

Comment:
I guess we are gonna do something in regards different assets for different devices later on... Do we have a ticket for that?

Reply:
Yes we do https://app.asana.com/0/1159164196409129/1192037358959244/f